packages/esm-patient-labs-app/src/lab-orders/add-lab-order/lab-order-form.component.tsx

```diff
@@ -60,90 +93,96 @@ export function LabOrderForm({ initialOrder, closeWorkspace }: LabOrderFormProps
             <Column lg={8} md={8} sm={4}>
               <InputWrapper>
-                <ComboBox
-                  size="lg"
-                  id="priorityInput"
-                  titleText={t('priority', 'Priority')}
-                  selectedItem={priorityOptions.find((p) => p.value == inProgressLabOrder?.urgency)}
-                  items={priorityOptions}
-                  itemToString={(item) => item?.label}
-                  onChange={({ selectedItem }) =>
-                    setInProgressLabOrder({
-                      ...inProgressLabOrder,
-                      urgency: selectedItem?.value ?? priorityOptions[0],
-                    })
-                  }
+                <Controller
+                  name="urgency"
+                  control={control}
+                  render={({ field: { onChange, onBlur, value } }) => (
+                    <ComboBox
+                      size="lg"
+                      id="priorityInput"
+                      titleText={t('priority', 'Priority')}
+                      selectedItem={priorityOptions.find((option) => option.value === value)}
+                      items={priorityOptions}
+                      onBlur={onBlur}
+                      onChange={({ selectedItem }) => onChange(selectedItem?.value || null)}
+                    />
+                  )}
                 />
+                <FormLabel className={styles.errorLabel}>{formState.errors.urgency?.message}</FormLabel>
               </InputWrapper>
             </Column>
           </Grid>
           <Grid className={styles.gridRow}>
             <Column lg={16} md={8} sm={4}>
               <InputWrapper>
-                <TextInput
-                  id="additionalInstructionsInput"
-                  size="lg"
-                  labelText={t('additionalInstructions', 'Additional instructions')}
-                  // placeholder={t('indicationPlaceholder', 'e.g. "Hypertension"')}
-                  value={inProgressLabOrder.instructions}
-                  onChange={(e) =>
-                    setInProgressLabOrder({
-                      ...inProgressLabOrder,
-                      instructions: e.target.value,
-                    })
-                  }
-                  maxLength={150}
+                <Controller
+                  name="instructions"
+                  control={control}
+                  render={({ field: { onChange, onBlur, value } }) => (
+                    <TextInput
+                      id="additionalInstructionsInput"
+                      size="lg"
+                      labelText={t('additionalInstructions', 'Additional instructions')}
+                      value={value}
+                      onChange={onChange}
+                      onBlur={onBlur}
+                      maxLength={150}
+                    />
+                  )}
                 />
+                <FormLabel className={styles.errorLabel}>{formState.errors.instructions?.message}</FormLabel>
```

Comment:
Why do we have a form label? For error messages in forms, we should use Inline Notifications. Please refer here: https://zeroheight.com/23a080e38/p/683580-notifications

![image](https://github.com/openmrs/openmrs-esm-patient-chart/assets/51502471/a392137c-5766-4d42-a2e4-39343bee8507)